src/lib/Scenes/Artwork/Components/__tests__/ArtworkAvailability-tests.tsx

```diff
@@ -0,0 +1,26 @@
+import { Sans } from "@artsy/palette"
+import { shallow } from "enzyme"
+import React from "react"
+import { ArtworkAvailability } from "../ArtworkAvailability"
+
+describe("ArtworkAvailability", () => {
+  it("renders buttons correctly", () => {
+    const component = shallow(<ArtworkAvailability artwork={artworkActionsArtwork} />)
+    expect(component.find(Sans).length).toEqual(1)
+
+    expect(
+      component
+        .find(Sans)
+        .at(0)
+        .render()
+        .text()
+    ).toMatchInlineSnapshot(`"On Loan"`)
+  })
+})
+
+const artworkActionsArtwork = {
```

Comment:
Do you need `__id` and `_id` here? Doesn't look like you're querying them from graphQL.

You can also rename this fixture as it refers to another component! Or... extract the fixture somewhere and share it between the two.

Reply:
let me check! i saw it complain about missing `__id` somewhere but maybe not here?